src/Service/ShipmentService.php

```diff
@@ -184,7 +184,12 @@ private function createNonDistributedShipment(Order $order, $idProduct, $isTestM
         $parcelWeight = 0;
         $orderShippingCost = $order->total_shipping_tax_incl ?: 0;
         foreach ($products as $product) {
-            $parcelPrice += round($product['total_wt'], 2);
+            if ($product['total_customization_wt'] && $product['total_customization_wt'] !== 0) {
```

Comment:
it looks like a lot of code is duplicated, maybe you can create a private function for that? Will be easier to read and change